RELEASE.md

```diff
@@ -12,6 +12,18 @@ This enables writing forward compatible code: by explicitly importing either `te
 * Adds `enable_tensor_equality()`, which switches the behavior such that: 
   * Tensors are no longer hashable.
   * Tensors can be compared with == and !=, yielding a Boolean Tensor with element-wise comparison results. This will be the default behavior in 2.0.
+* Auto Mixed-Precision graph optimizer simplifies converting models to float16 for acceleration on Volta and Turing Tensor Cores. This feature can be enabled by wrapping an optimizer class with `tf.train.experimental.enable_mixed_precision_graph_rewrite()`.
```

Comment:
Wasn't this in 1.14?

Reply:
yes, but it was never relnoted. I can begin the note with "Starting from TensorFlow 1.14 ..." if you like.

Reply:
I think this is fine. No need for the "Starting from TensorFlow 1.14"